syntax = "proto3";

package Packet;

message JoinGamePacket {
  string nickname = 1;
  string avatar_name = 2;
}